docs/cloud/pouta/launch-vm-from-web-gui.md

```diff
@@ -46,9 +46,7 @@ If you are already familiar with SSH keys, you can use your existing SSH keys to
 
 If you have not used SSH keypairs before, you need to create one. The web interface can take care of this for you. Go to **Key Pairs** section, select **Create Key Pair**. Give your key a name and click **Create**. You get a "_keyname.pem_" to save. Save it in your home directory.
 
-![ssh key pairs](/img/pouta-user-guide-keypairs.png)
-
-**Figure** The _Access & Security_ subpage in the cPouta web interface
```

Comment:
In this part, I have removed: **Figure** The _Access & Security_ subpage in the cPouta web interface